process/unsigned/interceptedUnsignedTransaction.go

```diff
@@ -14,15 +14,16 @@ import (
 
 // InterceptedUnsignedTransaction holds and manages a transaction based struct with extended functionality
 type InterceptedUnsignedTransaction struct {
-	uTx                      *smartContractResult.SmartContractResult
-	marshalizer              marshal.Marshalizer
-	hasher                   hashing.Hasher
-	addrConv                 state.AddressConverter
-	coordinator              sharding.Coordinator
-	hash                     []byte
-	rcvShard                 uint32
-	sndShard                 uint32
-	isAddressedToOtherShards bool
+	uTx          *smartContractResult.SmartContractResult
+	marshalizer  marshal.Marshalizer
+	hasher       hashing.Hasher
+	addrConv     state.AddressConverter
+	coordinator  sharding.Coordinator
+	hash         []byte
+	rcvShard     uint32
+	sndShard     uint32
+	isForMyShard bool
```

Comment:
I suggest renaming it to _isForCurrentShard_ but it's up to you

Reply:
Done